tests/foreman/api/test_lifecycleenvironment.py

```diff
@@ -198,3 +198,26 @@ def test_positive_search_in_org(self):
             {lc_env_.name for lc_env_ in lc_envs},
             {u'Library', lc_env.name},
         )
+
+    @tier2
+    @skip_if_bug_open('bugzilla', 1348727)
+    def test_verify_bugzilla_1348727(self):
```

Comment:
I tend to agree with @rochacbruno The name could follow our friendlier format AND have a BZ decorator pointed at the issue. Once the issue is VERIFIED the test will continue to mean something and the BZ naming wouldn't make sense anymore